android-reset-keystore.md

```diff
@@ -0,0 +1,19 @@
+# Reseting your Android Upload key
+## Background
+All newer apps (and many others) that are live on the Play Store are opted into [Google Play App Signing](https://support.google.com/googleplay/android-developer/answer/9842756?hl=en). With Play App Signing, Google maintains the signing key used to ultimately verify that your app is authentic. Meanwhile, when your app is built, it is signed with a separate key known as the "Upload key." Google Play checks your upload key when you go to publish your app to ensure the app binary is from you. If you lose this key, as long as you still have access to your Play Store account, you can reset it and continue publishing your app.
+
+## Reseting your key with EAS-managed credentials
+Google provides the most up-to-date information how to reset your key in their [Play Console Help section on Play App Signing](https://support.google.com/googleplay/android-developer/answer/9842756?visit_id=637973748658459850-3395295471&rd=1#reset), so be sure to consult that if you have any questions. However, if you use EAS to manage your Android app credentials, the process is generally a little easier, because the EAS CLI can create the new upload keystore for you.
+
+### Generate a new Upload keystore
+1. Run `eas credentials`.
+2. Choose "Android".
+3. Choose the correct build profile, generally your "production" build profile.
+4. Choose the "Keystore" option.
```

Comment:
```suggestion
4. Choose the "Keystore: Manage everything needed to build your project" option.
```

Let's the actual settings name from the CLI so that there is no confusion in future.